modules/ossm-deploy-cluster-wide-control-plane-cli.adoc

```diff
@@ -0,0 +1,73 @@
+////
+This module is included in the following assemblies:
+* service_mesh/v2x/ossm-create-smcp.adoc
+////
+:_content-type: PROCEDURE
+[id="ossm-deploy-cluster-wide-control-plane-cli_{context}"]
+= Configuring the control plane for cluster-wide deployment with the CLI
+
+You can configure the `ServiceMeshControlPlane` resource for cluster-wide deployment using the CLI. In this example, `istio-system` is the name of the Service Mesh control plane namespace.
+
+.Prerequisites
+
+* The {SMProductName} Operator must be installed.
+* Access to the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
++
+[source,terminal]
+----
+$ oc login --username=<NAMEOFUSER> https://<HOSTNAME>:6443
+----
++
+. Create a project named `istio-system`.
++
+[source,terminal]
+----
+$ oc new-project istio-system
+----
+
+. Create a `ServiceMeshControlPlane` file named `istio-installation.yaml` using the following example. 
++
+.Example version {MaistraVersion} istio-installation.yaml
+[source,yaml, subs="attributes,verbatim"]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  version: v{MaistraVersion}
+  mode: ClusterWide
+----
+
+. Run the following command to deploy the {SMProductShortName} control plane, where `<istio_installation.yaml>` includes the full path to your file.
++
+[source,terminal]
+----
+$ oc create -n istio-system -f <istio_installation.yaml>
+----
++
+. To watch the progress of the pod deployment, run the following command:
```

Comment:
```suggestion
. To monitor the progress of the pod deployment, run the following command:
```
- Would "monitor" be more accurate here? Your call.

Reply:
Done